scripts/memory_logger.py

```diff
@@ -1003,15 +1049,30 @@ def verifyArgs(args):
     else:
       args.outfile = [os.getcwd() + '/' + args.run[0].replace('..', '').replace('/', '').replace(' ', '_') + '.log']
 
-  if args.pstack:
-    if platform.platform(0, 1).split('-')[:-1][0].find('Darwin') != -1:
-      try:
-        import lldb
-      except ImportError:
-        print 'Unable to import lldb. The lldb API is now supplied by \nXcode but not automatically set in your PYTHONPATH. \nPlease search the internet for how to do this if you \nwish to use --pstack on Mac OS X.\n\nNote: If you installed Xcode to the default location of \n/Applications, you should only have to perform the following:\n\n\texport PYTHONPATH=/Applications/Xcode.app/Contents/SharedFrameworks/LLDB.framework/Resources/Python:$PYTHONPATH\n'
-        sys.exit(1)
+  if args.pstack and (args.read is None and args.plot is None):
+    if args.debugger is not None:
+      if args.debugger[0] == 'lldb':
+        if platform.platform().find('Darwin') != -1:
+          try:
+            import lldb
+          except ImportError:
+            print '\nUnable to import lldb\n\n\tIF USING MAC OS X; The Python lldb API is \
+now supplied by\n\tXcode but not automatically set in your PYTHONPATH. \n\tPlease search the \
+internet for how to do this if you \n\twish to use --pstack on Mac OS X.\n\n\tNote: If you \
+installed Xcode to the default location of \n\t/Applications, you should only have to perform \
+the following:\n\n\texport \
+PYTHONPATH=/Applications/Xcode.app/Contents/SharedFrameworks/LLDB.framework/Resources/Python:$PYTHONPATH \
+\n\n\t\t###!! IMPORTANT !!###\n\n\tIt may also be necessary to unload the miniconda module.\n\tIf \
+you receive a Fatal Python error about PyThreadState...\n\ttry using your systems version of \
+Python instead.\n\n'
```

Comment:
As @jwpeterson would say, this triggers my OCD. When you use the line continuation character you should still respect indentation. Python does the right thing.